Handlers/AuthHandlers.js

```diff
@@ -390,8 +495,8 @@ function forgotPassword (req, res, next) {
         function (callback) {
             if (sendLink != undefined && sendLink == "true") {
                 //Send verification email 
-                var showroomURL = "https://" + config.SHOWROOM_HOST + "/changePassword"; 
-                var subject = "Reset your password"
+                var showroomURL = "https://" + config.SHOWROOM_HOST + "/changePassword/" + userID + "/" + emailUUID; 
```

Comment:
if testing in a local environment hardcoding https will bring issues. make a function change between http and https depending on config variable